app/services/Athena.scala

```diff
@@ -2,8 +2,6 @@ package services
 
 import com.typesafe.scalalogging.StrictLogging
 import io.circe.{Decoder, Encoder}
-import models.GroupedVariantViews.VariantName
```

Comment:
Note that this Athena file is still needed for the Super Mode page, which also queries athena data. I'll deal with this separately